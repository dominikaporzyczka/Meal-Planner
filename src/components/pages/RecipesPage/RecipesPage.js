import React, { Component } from 'react';

export class RecipesPage extends Component {
  render() {
    return (
      <div>RecipesPage</div>
    );
  }
};
