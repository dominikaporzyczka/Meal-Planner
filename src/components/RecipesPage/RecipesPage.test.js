import React from 'react';
import renderer from 'react-test-renderer';
import { RecipesPage } from './RecipesPage.js';

describe('RecipesPage', () => {
  it('should match snapshot', () => {
    const recipesPage = renderer
      .create(<RecipesPage />)
      .toJSON();

    expect(recipesPage).toMatchSnapshot();
  });
});
