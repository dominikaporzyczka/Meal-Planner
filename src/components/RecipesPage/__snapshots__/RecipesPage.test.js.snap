// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipesPage should match snapshot 1`] = `
<div>
  RecipesPage
</div>
`;
